docs/book/api-docs/oss-api/oss-api/getting-started.md

```diff
@@ -0,0 +1,141 @@
+---
+icon: person-from-portal
+---
+
+# Getting Started
+
+The ZenML OSS server is a FastAPI application, therefore the OpenAPI-compliant docs are available at `/docs` or `/redoc` of your ZenML server:
+
+{% hint style="info" %}
+In the local case (i.e. using `zenml login --local`, the docs are available on `http://127.0.0.1:8237/docs`)
+{% endhint %}
+
+![ZenML API docs](../../../.gitbook/assets/zenml_api_docs.png)
+
+![ZenML API Redoc](../../../.gitbook/assets/zenml_api_redoc.png)
```

Comment:
Add a bit of text here about what the redoc version is? just to make it look a bit nicer instead of just two images next to one another?